README.md

```diff
@@ -150,100 +151,140 @@ Now, you can use these dashboards to monitor RabbitMQ and Postgres metrics.
 
-This can be done by running the following command:
+The following command will process the tv shows:
 
 ```
-docker exec -it knightcrawler-postgres-1 psql -d knightcrawler -c "
+docker exec -it knightcrawler-postgres-1 psql -U postgres -d knightcrawler -c "
 INSERT INTO ingested_torrents (name, source, category, info_hash, size, seeders, leechers, imdb, processed, \"createdAt\", \"updatedAt\")
 SELECT title, 'RARBG', 'tv', hash, size, NULL, NULL, imdb, false, current_timestamp, current_timestamp
 FROM items where cat LIKE 'tv%%' ON CONFLICT DO NOTHING;"
 ```
 
-And similarly for movies:
+This will do the movies:
+
 ```
-docker exec -it knightcrawler-postgres-1 psql -d knightcrawler -c "
+docker exec -it knightcrawler-postgres-1 psql -U postgres -d knightcrawler -c "
 INSERT INTO ingested_torrents (name, source, category, info_hash, size, seeders, leechers, imdb, processed, \"createdAt\", \"updatedAt\")
 SELECT title, 'RARBG', 'movies', hash, size, NULL, NULL, imdb, false, current_timestamp, current_timestamp
 FROM items where cat LIKE 'movies%%' ON CONFLICT DO NOTHING;"
 ```
 
-You should get a response similar to:
+Both should return a response that looks like:
+
+```
+INSERT 0 669475
+```
 
-`INSERT 0 669475`
+As soon as both commands have been run, we can immediately drop the items table. The data inside has now been processed, and will just take up space.
 
-After, you can delete the `items` table by running:
+```
+docker exec -it knightcrawler-postgres-1 psql -U postgres -d knightcrawler -c "drop table items";
+```
 
-```docker exec -it knightcrawler-postgres-1 psql -d knightcrawler -c "drop table items";```
 
+### I imported the data without the `LIKE 'movies%%'` queries!
 
-#### Fixing imported databases
+If you've already imported the database using the old instructions, you will have imported a ton of new movies and tv shows, but the new queries will give you a lot more!
 
-If you've already imported a database with incorrect categories, you can fix these categories by running the following commands:
+Don't worry, we can still correct this. We can retroactively update the categories by running the following commands
 
-```docker exec -it knightcrawler-postgres-1 psql -d knightcrawler -c "UPDATE ingested_torrents  SET category='movies', processed='f' WHERE category LIKE 'movies_%';"```
+```
+# Update movie categories
+docker exec -it knightcrawler-postgres-1 psql -d knightcrawler -c "UPDATE ingested_torrents  SET category='movies', processed='f' WHERE category LIKE 'movies_%';"```
 
-```docker exec -it knightcrawler-postgres-1 psql -d knightcrawler -c "UPDATE ingested_torrents  SET category='movies', processed='f' WHERE category LIKE 'movies_%';"```
+# Update TV categories
+docker exec -it knightcrawler-postgres-1 psql -d knightcrawler -c "UPDATE ingested_torrents  SET category='movies', processed='f' WHERE category LIKE 'movies_%';"
```

Comment:
this got merged with my error, should be

```diff
- docker exec -it knightcrawler-postgres-1 psql -d knightcrawler -c "UPDATE ingested_torrents  SET category='movies', processed='f' WHERE category LIKE 'movies_%';"
+ docker exec -it knightcrawler-postgres-1 psql -d knightcrawler -c "UPDATE ingested_torrents  SET category='tv', processed='f' WHERE category LIKE 'tv_%';"
```

Reply:
Great! Someone else needs to push it or I’ll do it in a few hours 😀